source/Libraries/UplayLibrary/UplayMetadataProvider.cs

```diff
@@ -55,7 +55,7 @@ public override GameMetadata GetMetadata(Game game)
             }
             else
             {
-                var program = Programs.GetUnistallProgramsList().FirstOrDefault(a => a.RegistryKeyName == "Uplay Install " + game.GameId);
+                var program = Programs.GetUnistallProgramsList().FirstOrDefault(a => a.RegistryKeyName == "Ubisoft Connect Install " + game.GameId);

```

Comment:
I don't think this is correct. Last I checked they still used `Uplay` in the string for backwards compatibility. Please make sure this is actually correct change.

Reply:
You're right, fixed.
Tested out by commenting the `if` part with both names, only that got me to this part of code.